pkg/provider/github/detect_test.go

```diff
@@ -59,12 +59,11 @@ func TestProvider_Detect(t *testing.T) {
 			processReq: true,
 		},
 		{
-			name: "unsupported Event",
```

Comment:
maybe still have a unit test for unsupported event?

Reply:
added